src/vmspawn/vmspawn.c

```diff
@@ -158,6 +190,27 @@ static int parse_argv(int argc, char *argv[]) {
                             return log_error_errno(r, "Failed to parse --qemu-kvm=%s: %m", optarg);
                         break;
 
+                case ARG_QEMU_VSOCK:
+                        r = parse_tristate(optarg, &arg_qemu_vsock);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse --qemu-vsock=%s: %m", optarg);
+                        break;
+
+                case ARG_VSOCK_CID: {
+                        unsigned int cid;
```

Comment:
in our codebase we typically spell this just as `unsigned`, not as `unsigned int`, but doesn't really matter

Reply:
Fixed in ef3bb39e80.